auth/static.go

```diff
@@ -38,6 +38,14 @@ func ValidateHeader(header *Header) error {
 	if header == nil {
 		return consts.ErrNilHeader
 	}
+	tokenType := header.TokenTyp
+	if tokenType < NoType || tokenType > Jwt {
```

Comment:
There are some new tests. Is it necessary for me to test these out too when I am using this method (and others) from `hwsc-user-svc`?

Reply:
Unnecessary